### ISSUE TITLE
Switching to systemd instructions are not in the wiki 

### ISSUE THREAD

**Issue by mozgwar** [gordonb3/bubbagen#5]
The instructions on how to switch to systemd are not on the wiki

**Comment by gordonb3** [comment]
Yes there were some issues with the original instructions as vital services enabled under the original initv installation ended up not being enabled under systemd, so I had to retract them.
